packages/internals/src/__tests__/getGenerators/getGenerators.test.ts

```diff
@@ -381,8 +381,7 @@ describe('getGenerators', () => {
   })
 
   it('basic - binaryTargets as env var - darwin, windows, debian', async () => {
-    process.env.BINARY_TARGETS_ENV_VAR_TEST =
```

Comment:
Why the removal of these?

Reply:
oops maybe I made mistake on rebase. fixed.